PhysicsTools/PatAlgos/python/slimming/prunedGenParticles_cfi.py

```diff
@@ -5,13 +5,17 @@
     select = cms.vstring(
         "drop  *", # this is the default
         "++keep abs(pdgId) == 11 || abs(pdgId) == 13 || abs(pdgId) == 15", # keep leptons, with history
-        "keep abs(pdgId) == 12 || abs(pdgId) == 14 || abs(pdgId) == 16",   # keep neutrinos
         "drop   status == 2",                                              # drop the shower part of the history
+        "keep++ (400 < abs(pdgId) < 600) || (4000 < abs(pdgId) < 6000)",   # keep decays for BPH studies
```

Comment:
hi @alberto-sanchez - for my information what does keep vs keep+ vs keep++ mean in this syntax?